src/views/portals/YetiView.vue

```diff
@@ -504,7 +515,9 @@
         mountains: {},
         visibleMountains: {},
         promiseMountains: null,
-        showMountainsList: false
+        showMountainsList: false,
+
+        documents: null
```

Comment:
I see that you use `documents` only for map view, and several times as `documents[0]`. Would it be preferable to have `document`property, and call `map-view` with `[document]` ?